app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
-  belongs_to :users
-  belongs_to :posts
+  belongs_to :user
+  belongs_to :post
```

Comment:
- [ ] [REQUIRED] Good job so far! The associations between models should be set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), please implement it like that 🙏🏼 

```ruby
belongs_to :author, class_name: 'User'
belongs_to :post
```

Reply:
@mukhammadrizooff Thanks for the suggestion